packages/rum-core/src/domain/view/trackViews.spec.ts

```diff
@@ -317,6 +317,22 @@ describe('view lifecycle', () => {
       expect(getViewCreateCount()).toEqual(1)
     })
   })
+
+  it('should notify VIEW_CREATED before VIEW_CREATED', () => {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
```suggestion
  it('should notify BEFORE_VIEW_CREATED before VIEW_CREATED', () => {
```